addons/microsoft_calendar/models/calendar.py

```diff
@@ -237,10 +237,12 @@ def action_mass_archive(self, recurrence_update_setting):
 
     def _get_microsoft_sync_domain(self):
         # in case of full sync, limit to a range of 1y in past and 1y in the future by default
+        # decreasing the 'range_lower_bound' param you can limit old events updates in Odoo and avoid email spam on Microsoft
         ICP = self.env['ir.config_parameter'].sudo()
-        day_range = int(ICP.get_param('microsoft_calendar.sync.range_days', default=365))
-        lower_bound = fields.Datetime.subtract(fields.Datetime.now(), days=day_range)
-        upper_bound = fields.Datetime.add(fields.Datetime.now(), days=day_range)
+        lower_bound_day_range = int(ICP.get_param('microsoft_calendar.sync.range_lower_bound', default=365))
+        upper_bound_day_range = int(ICP.get_param('microsoft_calendar.sync.range_upper_bound', default=365))
+        lower_bound = fields.Datetime.subtract(fields.Datetime.now(), days=lower_bound_day_range)
+        upper_bound = fields.Datetime.add(fields.Datetime.now(), days=upper_bound_day_range)
```

Comment:
we should still manage microsoft_calendar.sync.range_days because people who defined it expect it work.
You should reuse microsoft_calendar.sync.range_days  for the upper bound and define a lower bound value. That way, people who defined microsoft_calendar.sync.range_days keep their behavior: symetric behavior. We don't break anything as there as drawback to work like this.